src/backend/distributed/citus.control

```diff
@@ -1,6 +1,6 @@
 # Citus extension
 comment = 'Citus distributed database'
```

Comment:
minor: we typically also bump the version in `sql/multi_extension.sql` regression test. I think it might even make sense to simply call these two functions 

```SQL
ALTER EXTENSION citus UPDATE TO '8.4-1';
.. some new lines to allow other future ALTER EXTENSIONs


-- just call the functions for sanity check, 
-- showing that nothing errors out is good enough for now
SELECT citus_prepare_pg_upgrade();
SELECT citus_finish_pg_upgrade();

```

Reply:
Ah, this probably wouldn't work because the trigger creation and setting the dependency would fail as they are already there.

Reply:
Thanks for pointing out `sql/multi_extension.sql`, will add the bump there.

Great thinking about the invocation of the functions, I will see if and how they will fail if you call them outside of an upgrade. My guess is that you are correct and it will fail on the triggers and dependencies.

We need to come back to these changes for testing in the short future anyway. Hanefi will send some pointers next week on how he tested the upgrades. Until then we will need to keep following our old testing documents to verify it works.

Reply:
calling the functions without upgrading causes even earlier failures:

```
postgres=# SELECT citus_prepare_pg_upgrade();
┌──────────────────────────┐
│ citus_prepare_pg_upgrade │
├──────────────────────────┤
│                          │
└──────────────────────────┘
(1 row)

Time: 29.438 ms
postgres=# SELECT citus_finish_pg_upgrade();
ERROR:  duplicate key value violates unique constraint "pg_dist_node_pkey"
DETAIL:  Key (nodeid)=(1) already exists.
CONTEXT:  SQL statement "INSERT INTO pg_catalog.pg_dist_node SELECT * FROM public.pg_dist_node"
PL/pgSQL function citus_finish_pg_upgrade() line 14 at SQL statement
Time: 6.338 ms
```

Don't think it is good to drop all the things we currently replicate to test these functions, rather have a good upgrade test that will be easier to maintain.